docs/404.html

```diff
@@ -0,0 +1,23 @@
+---
```

Comment:
This was created via Jekyll when I created a project locally, and it seems like a useful file to keep around.